tests/src/test/scala/cats/tests/OneAndTests.scala

```diff
@@ -16,6 +16,7 @@ class OneAndTests extends CatsSuite {
   checkAll("OneAnd[Stream, Int]", OrderLaws[OneAnd[Stream, Int]].eqv)
 
   checkAll("OneAnd[Stream, Int] with Option", TraverseTests[OneAnd[Stream, ?]].traverse[Int, Int, Int, Int, Option, Option])
+  checkAll("OneAnd[Stream, Int] with Option", Traverse1Tests[OneAnd[Stream, ?]].traverse1[Option, Int, Int, Int, Int, Option, Option])
```

Comment:
The old `Traverse[OneAnd[F, ?]]` instance is not tested anymore.

Can you add something like

```scala
{
  implicit val traverse = ListWrapper.traverse
  checkAll("OneAnd[ListWrapper, Int] with Option", TraverseTests[OneAnd[LisTWrapper, ?]].traverse[Int, Int, Int, Int, Option, Option])
  checkAll("Traverse[OneAnd[ListWrapper, A]]", SerializableTests.serializable(Traverse[OneAnd[ListWrapper, ?]]))
}
```

Can you also add `SerializableTests` for `Traverse1` here (`OneAnd`) and for `NonEmptyList` and `NonEmptyVector`?

You can also remove the tests for `Traverse` (line 18 and 20 here) and for `NonEmptyList` and `NonEmptyVector` as those tests are covered by the `Traverse1Tests` you wrote as well.